src/navigations/HomeBottomTabs.js

```diff
@@ -1,11 +1,9 @@
 /* eslint-disable no-use-before-define */
 import * as React from 'react';
-import PushNotification from 'react-native-push-notification';
-import PushNotificationIOS from '@react-native-community/push-notification-ios';
-import messaging from '@react-native-firebase/messaging';
-import {Platform, SafeAreaView, StyleSheet, View} from 'react-native';
+import {SafeAreaView, StyleSheet, View} from 'react-native';
 import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
-import {useRecoilState, useRecoilValue} from 'recoil';
+import {useNavigation} from '@react-navigation/core';
+import {useRecoilValue} from 'recoil';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been modified. The push notification related imports have been removed and replaced with `usePushNotificationHook`. This change is in line with the PR description which mentions refactoring of push notification handling.
```diff
- import PushNotification from 'react-native-push-notification';
- import PushNotificationIOS from '@react-native-community/push-notification-ios';
- import messaging from '@react-native-firebase/messaging';
+ import {useRecoilValue} from 'recoil';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->